go.mod

```diff
@@ -1,9 +1,13 @@
-module github.com/artyom/leproxy
+module github.com/cubewise-tryan/leproxy
```

Comment:
are you going to change module name in the main repo?

Reply:
I have made lots of changes since and created a fork as I thought this project was dead. New one is here: https://github.com/cubewise-code/sweetssl

Reply:
So this pr is outdated?